### ISSUE TITLE
Cross Validation

### ISSUE THREAD

**Issue by dee-hub** [Abdulbaqqi/Breast-cancer#2]
Cross validation should be done on your validation set not on your train set. Validating your data on the same training set will give you a very high accuracy which is actually unrepresentative.
